README.md

```diff
@@ -1 +1,27 @@
-# docs
\ No newline at end of file
+Cairo 92
+========
+
+Este documento tem por objetivo registrar decisões tomadas em relação a administração do co-working Cairo 92. Cabe aos membros associados decidir sobre qualquer situação que não esteja aqui descrita.
+
+## Despesas mensais
+
+O período no qual as despesas mensais são catalogadas e divididas entre os atuais membros presentes no co-working compreende o primeiro ao último dia do mês. Os pagamentos são realizados no mês seguinte ao mês catalogado.
+
+O pagador de uma determinada despesa deve ter em mente que, caso esteja com saldo positivo, apenas receberá o reembolso deste valor no mês seguinte à despesa. Caso o responsável não deseje ou possa arcar com a despesa, é sugerido que converse com os demais membros, de preferência os associados, para se encontrar uma forma de efetuar a compra ou pagamento pendente. Há ainda a possibilidade de se utilizar o caixa do co-working para pagamento de uma despesa, em casos decididos pelos associados.
+
+### Catálogo de despesas
+
+A atual ferramenta utilizada para catalogar as despesas é o [SplitWise][splitwise]. Todos os membros do co-working tem permissão de leitura e escrita, mas se faz necessário que alterações sejam realizadas por dois ou mais membros, a fim de diminuir a chance de ocorrerem erros no processo.
```

Comment:
Sempre requerer 2 pessoas pra inserir itens pode ficar meio complicado. Se todo mundo instalar o aplicativo e receber as notificações, a gente pode sempre revisar o que for acontecendo e acredito que funcionaria bem.

Edições em itens já colocados realmente seria interessante haver uma comunicação mais explícita, talvez uma mensagem no slack com uma explicação.

Reply:
Acredito que a mensagem no Slack, mesmo desacompanhado já aliviaria bem as coisas. Ou a integração entre SpliWise e Slack https://zapier.com/apps/splitwise/integrations/slack

Reply:
Sugeri os anúncios no `#tesouraria`. Vou trocar a redação para:

```suggestion
A atual ferramenta utilizada para catalogar as despesas é o [SplitWise][splitwise]. Todos os membros do co-working tem permissão de leitura e escrita, mas se faz necessário que alterações sejam previamente informadas no canal `#tesouraria` do Slack, a fim de diminuir a chance de ocorrerem erros no processo.
```